packages/ui/src/components/link/Link.component.js

```diff
@@ -0,0 +1,17 @@
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+import * as colors from '../../colors';
+
+export const Link = styled.a.attrs({ target: '_blank', rel: 'noreferrer' })`
+  color: ${colors.B1};
+
+  &:hover {
+    color: #3739d2
+  }
+`;
```

Comment:
The hyperlink styling is more like a part of global reset stylings, so we could consider putting it into [createAppGlobalStyle](https://github.com/pingcap/tug-website/blob/main/packages/ui/src/index.js#L19).

But if we're concerned that the global stylings may break other pages, we could consider reset the hyperlink in a page-level container like:

```js
// In src/pages/account/organization/new/index.styled.js

const Container = styled.div`
  a {
    color: ${colors.B1};
    
    &:hover {
      color: #3739d2
    }
  }
`;
```

Reply:
I think overriding `<a>` styles on it's container is not a good choice, It will lead to some critical style issues. The `Link` component I wrapped is only represent the highlighting jumping links, so I will keep it until we found a more reasonable solution.